js/src/ui/publishBox.js

```diff
@@ -0,0 +1,74 @@
+var contentScoreName = 'content';
+var keywordScoreName = 'keyword';
+var notAvailableStatus = 'na';
+var scoreDescriptionClass = 'score-text';
+var imageScoreClass = 'image yoast-logo svg';
+
+(function( $ ) {
+	'use strict';
+	/**
+	 * Converts the first letter to uppercase in a string.
```

Comment:
Please add a whitespace between the function description and the parameters / return values.